src/scan.sh

```diff
@@ -32,36 +31,36 @@ cd $SHARED_DIR/artifacts/ || exit
 # was not initially a concern:
 # https://aws.amazon.com/blogs/compute/announcing-improved-vpc-networking-for-aws-lambda-functions/
 /home/scanner/domain-scan/scan $SHARED_DIR/artifacts/scanme.csv \
```

Comment:
The `/home/scanner` path is no longer valid in this PR. If you want to use absolute paths you should probably leverage the `CISA_HOME` environment variable created in the Dockerfile. You will also need to edit [this line in the cisagov/orchestrator Compose file](https://github.com/cisagov/orchestrator/blob/4b573ad8205dcf17cc907eab969d96c4ddd8b20b/docker-compose.yml#L42).

Reply:
See commit 8b06a9d.

Reply:
I'm not a fan of hard-coding `/home/cisa` since the unprivileged user's username is a build argument. I know it does not impact our base image but if I build the Docker image locally and pass in `--build-args CISA_USER=testing` then the appropriate path in the image will be `/home/testing`. We set the `CISA_HOME` environment variable in the Dockerfile and it is thus available inside the image. I would prefer we use this environment variable to ensure it is always using the correct absolute path.

Reply:
Please see commit d468a3c.